lib/cartopy/tests/mpl/test_gridliner.py

```diff
@@ -17,8 +17,13 @@
 import cartopy.crs as ccrs
 from cartopy.mpl import _MPL_36
 from cartopy.mpl.geoaxes import GeoAxes
-from cartopy.mpl.gridliner import (LATITUDE_FORMATTER, LONGITUDE_FORMATTER, Gridliner,
-                                   classic_formatter, classic_locator)
+from cartopy.mpl.gridliner import (
+    LATITUDE_FORMATTER,
+    LONGITUDE_FORMATTER,
+    Gridliner,
+    classic_formatter,
+    classic_locator,
+)
```

Comment:
I think I prefer the old style here, as I like the horizontal separation between modules and their objects.  I can't see a setting for that though, and it isn't a deal-breaker.

Reply:
I don't have a strong preference either way, so I was fine applying this. I think that ruff likely takes the black formatting approach and applies it liberally here even if we aren't using black. I also haven't found a setting to override this so I think we are stuck with it if we want to replace isort with ruff.